modules/protobuf/27.0-rc2/presubmit.yml

```diff
@@ -0,0 +1,25 @@
+matrix:
+  platform: ["debian10", "macos", "ubuntu2004", "windows"]
+
+tasks:
+  verify_targets:
+    name: "Verify build targets"
```

Comment:
@zhangskz @googleberg All presubmit tasks now need to specify compatible Bazel versions, see https://github.com/bazelbuild/bazel-central-registry/blob/main/docs/README.md#anonymous-module-test. You can make changes to this file directly on this PR, no need for a new release.

Reply:
Thanks - done!